### PR TITLE
📝 (Site Completeness): Clarify need for no index and basic auth

### PR DESCRIPTION
**Pull request by cromwellryan**

### CODE REVIEW

File: project_management/site_completeness_checklist.md [hunk 1]
```diff
@@ -5,16 +5,14 @@ This list uses [Github Checklists](https://github.com/blog/1375-task-lists-in-gf
 
 ## Early Dev Checklist 
 
-* [ ] Dev site disallows indexing
-* [ ] Other item
-* [ ] all states of navigation/buttons (active, hover, inactive, etc)
-* [ ] dropdown menus
-* [ ] pagination
-* [ ] headline styles that work when very long (wrap to multiple lines)
```
Comment (nrambeck) [inline project_management/site_completeness_checklist.md]: Are we intending to remove these 4 design items? Not sure these really fall into site completeness anyway.

File: project_management/site_completeness_checklist.md [hunk 2]
```diff
@@ -5,16 +5,14 @@ This list uses [Github Checklists](https://github.com/blog/1375-task-lists-in-gf
...
-* [ ] Other item
-* [ ] all states of navigation/buttons (active, hover, inactive, etc)
-* [ ] dropdown menus
-* [ ] pagination
-* [ ] headline styles that work when very long (wrap to multiple lines)
+### Hosting
+* [ ] Automated Deployments
+* [ ] Preproduction Environments: Disallows indexing
+* [ ] Preproduction Environments: Require Basic Authentication
+
```
Comment (nrambeck) [inline project_management/site_completeness_checklist.md]: Remove this line break and add one below Hosting.

File: project_management/site_completeness_checklist.md (hunk 1 above)
Comment (cromwellryan) [inline project_management/site_completeness_checklist.md]: That was my plan, yeah.

**Comment by cromwellryan** [comment]
@nrambeck fixed the line endings and replied (twice) to your question 😄 . I've also rebased.

**Pull request merged**